apps/examples/testcase/ta_tc/media/itc/itc_media_fileinputdatasource.cpp

```diff
@@ -0,0 +1,87 @@
+/****************************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <media/FileInputDataSource.h>
+#include "tc_common.h"
+
+/****************************************************************************
+ * Definitions
+ ****************************************************************************/
+#define MEDIA_TEST_FILE_PATH "/mnt/fileinputdatasource.raw";
+#define FILE_DEFAULT_RATE 16000
+
+/****************************************************************************
+ * Global Variables
+ ****************************************************************************/
+static const char file_path[] = MEDIA_TEST_FILE_PATH;
+static const char test_data[] = "dummydata";
+static unsigned char *buf;
+
+static void SetUp(void)
```

Comment:
void -> int and check the success case value in main call.

Reply:
used a global flag